src/components/BlockingViews/BlockingView.js

```diff
@@ -56,7 +57,10 @@ const BlockingView = (props) => (
             height={props.iconHeight}
         />
         <Text style={[styles.notFoundTextHeader]}>{props.title}</Text>
-        <Text style={[styles.textAlignCenter]}>{props.subtitle}</Text>
+        <AutoEmailLink
```

Comment:
@allroundexperts what is the need for this change?

Reply:
Pardon me for my ignorance, but can you please share the context of why this was needed? I cannot find it in the issue comments?

Reply:
Neither I can find the screenshots/test steps related to that.

Reply:
Cool. Thanks for clarifying.

Reply:
Based on the comment, I think it would be better to add a description at the top of the file.